tests/security/yast2_apparmor/settings_disable_enable_apparmor.pm

```diff
@@ -47,6 +47,8 @@ sub run {
     send_key "alt-e";
     assert_screen("AppArmor-Settings-Enable-Apparmor");
     wait_screen_change { send_key "alt-q" };
+    clear_console;
```

Comment:
No need for clear_console.

Reply:
Agree with you in some degree but I would like to keep it. (it is noharm and we can get a clean screen)